EndlessClient/Dialogs/ListDialogItem.cs

```diff
@@ -18,6 +18,10 @@ public enum ListItemStyle
             Large
         }
 
+
+        public bool EnableBarberMode = false;
```

Comment:
`ListDialogItem` should not have anything barber-specific. This is a general control for all list dialog items. Also, convention is to avoid public fields and use properties instead.

If you want to override the width of the highlight, add a property that sets it, similar to `OffsetX` and `OffsetY`.

`OffsetX` already performs the function you're doing in the `Draw` override for getting `rectX`.